src/test/kotlin/com/github/avrokotlin/avro4k/schema/PolymorphicClassSchemaTest.kt

```diff
@@ -43,20 +38,6 @@ class PolymorphicClassSchemaTest : StringSpec({
       schema shouldBe expected
    }
 
-   "polymorphic defaults are not supported" {
-      val module = SerializersModule {
-         polymorphic(UnsealedPolymorphicRoot::class) {
-            subclass(UnsealedChildOne::class)
-            subclass(UnsealedChildTwo::class)
-            default { TypeSerializer }
-         }
-      }
-      val exception = shouldThrow<SerializationException> {
-         Avro(serializersModule = module).schema(UnsealedPolymorphicRoot.serializer())
-      }
-      exception.message shouldBe "Polymorphic defaults are not supported in avro4k. Error whilst describing: UnsealedPolymorphicRoot"
-   }
-
```

Comment:
You don't think it's worth keeping this test kind of as documentation - in case people come looking trying to figure out if this is supported or not. Not strong feelings either way, though

Reply:
I removed the exception if the SerializersModule contained a default subclass for polymorphic classes. Although default subclasses do not make sense for avro, I don't want to limit the reusability of SerializersModules across serialization formats. So I think the test does not make that much sense without the assertion for an exception.